Library/Homebrew/cmd/test-bot.rb

```diff
@@ -244,22 +241,9 @@ def safe_formulary(formula)
     end
 
     def git(*args)
-      rd, wr = IO.pipe
-
-      pid = fork do
-        rd.close
-        STDERR.reopen("/dev/null")
-        STDOUT.reopen(wr)
-        wr.close
-        Dir.chdir @repository
-        exec("git", *args)
+      Dir.chdir(@repository) do
+        Utils.popen_read("git", *args)
       end
```

Comment:
I revert back to use `Pathname#cd` as I found not of all git commands work well with `-C <path>` argument.